### PR TITLE
Fix login

### PR DESCRIPTION
**Pull request by mfka**

### CODE REVIEW

**Comment by mfka** [comment]
Finally it is done. Works like before - there are still some errors- i have not time to repair everything ;(

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (plorencrstit) [inline .eslintrc]: 👍

File: src/App.vue [hunk 2]
```diff
@@ -1,28 +1,28 @@
 <template>
   <div id="app">
-    <Header></Header>
+    <Header />
+    <Notification />
```
Comment (plorencrstit) [inline src/App.vue]: 👍

File: src/components/Alerts/Error.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<template>
+  <b-message v-if="content || title" :title="title" type="is-danger">{{ content }}</b-message>
+</template>
+<script lang="ts">
```
Comment (plorencrstit) [inline src/components/Alerts/Error.vue]: 👍

File: src/components/Breadcrumbs.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<template>
```
Comment (plorencrstit) [inline src/components/Breadcrumbs.vue]: 👍

**Comment by plorencrstit** [comment]
@mfka what does not work at all/properly in your opinion? Do you have any list?

**Pull request merged**